README.md

```diff
@@ -114,7 +118,7 @@ This is a requirement because receiving multicast datagrams requires a
 [level 2 multicast conformant](https://www.tldp.org/HOWTO/Multicast-HOWTO-2.html#ss2.2)
 socket API.
 The required multicast socket options and constants have been added with
-[PHP 5.4](http://php.net/manual/en/migration54.global-constants.php) (and up).
+[PHP 5.4](https://php-legacy-docs.zend.com/manual/php5/en/migration54.new-features) (and up).
```

Comment:
Good catch, thanks for looking into this and for going the extra mile to report upstream! In this particular case, I think we can safely remove the entire link as it doesn't provide a lot of value anymore nowadays. If PHP decides old documentation should no longer be hosted on their end, I would rather avoid linking to the Zend version (another external dependency that may break in the future, plus it happens to be commercial and somewhat slow).

May I ask you to update/reword this to avoid the link? (Also note the same text in `Factory.php`)

Reply:
Thank you @clue for your input here. After discussing the wording with the help of @cassifyit we changed it into the sentence: 

> The required multicast socket options and constants have been added and
> are only available to the low level socket API (ext-sockets), not
> to the newer stream based networking API.

Should be better after all. 👍

Reply:
Hm. I think there's still value in stating this does in fact require PHP 5.4+. The updated sentence seems to be missing this now and appears a bit clumsy imho. Perhaps you can rephrase this to improve clarity? 👍